README.md

```diff
@@ -128,17 +129,31 @@ In addition, the provider can be configured using attributes in the
 configuration. Attributes specified in `main.tf` override the configuration loaded by
 `conjur-api-go`.
 
-For example, if the environment is initialized as above, this configuration would
-authenticate as `terraform-user` instead of `admin`:
+For example, with `conjur_api_key` and `conjur_ssl_cert`defined as
+[input variables](https://www.terraform.io/docs/configuration/variables.html), this
+type of configuration could be used:
 
 ```
 # main.tf
+variable "conjur_api_key" {}
+variable "conjur_ssl_cert" {}
+
 provider "conjur" {
+  appliance_url = "http://conjur-server"
+  ssl_cert = var.conjur_ssl_cert
+  # If you have the certificate as a file, use this line instead
+  # ssl_cert_path = var.conjur_ssl_cert_path
+
+  account = "myorg"
+
   login = "terraform-user"
```

Comment:
maybe the login above should be `terraform-user` too? as you can probably tell, it bugs me that the values are different in this example than in the one above :)